### DISCUSSION TITLE
DAGs Stuck in Queued State with Google Composer and Airflow 2.5.3

### DISCUSSION THREAD

**Discussion by jorgeortizfuentes** [apache/airflow#34974] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Hi, I am running Airflow 2.5.3 on Google Composer 2.4.5, and I am experiencing issues with my DAGs.

All my DAGs are stuck in the queued state, including a test one that does nothing but print the current time.

Previously, I was running Airflow on a machine within a Kubernetes cluster, and the DAGs were functioning without any problems. Occasionally, I had issues and would resolve them by restarting the pods, which would then get everything back on track.

Now, on Google Composer, I've attempted to restart the Scheduler using the following command:

```
kubectl rollout restart deployment airflow-scheduler -n composer-2-4-5-airflow-2-5-3-6623aac0
```

However, this approach hasn't resolved the issue. The DAGs remain stuck in the queued state. I've also tried pausing and removing all DAGs, but to no avail.

### How to reproduce

I only installed Airflow through Google Composer.

### Operating System

Google Cloud Composer

### Deployment

Google Cloud Composer

### Deployment details

Resources
Workloads configuration
Scheduler
2 schedulers with 1 vCPU, 2 GB memory, 1 GB storage each
Triggerer
2 triggerers with 0.5 vCPU, 0.5 GB memory, 1 GB storage each
Web server
1 vCPU, 2 GB memory, 1 GB storage
Worker
Autoscaling between 1 and 8 workers, with 1 vCPU, 2 GB memory, 1 GB storage each
Core infrastructure
Environment size
Medium

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by eladkal** [comment]
Queue state issues were mostly resolved with Airflow 2.6.0
I suggest to update Airflow version first.

**Comment by Taragolis** [comment]
This issue/troubleshooting related to managed service, if you can't upgrade version of Airflow which suggested by @eladkal (I guess 2.5.3 it is last available version in [Cloud Composer](https://cloud.google.com/composer/docs/concepts/versioning/composer-versions)) you've better contact with GCP support. Airflow Community have limited knowledge about any managed services around AIrflow 😞

**Comment by mehdigati** [comment]
First, check if you have reached the maximum number of workers allowed in your Google Composer environment. You can try deleting the existing workers from the GKE cluster. Composer will automatically spin up new workers to handle the workload. If the above steps don't resolve the issue, you may want to consider upgrading to a newer version of Apache Airflow.
